vertica/datadog_checks/vertica/vertica.py

```diff
@@ -47,30 +44,9 @@ def __init__(self, name, init_config, instances):
 
         self._client_lib_log_level = self.instance.get('client_lib_log_level', self._get_default_client_lib_log_level())
 
-        self._tls_verify = is_affirmative(self.instance.get('tls_verify', False))
```

Comment:
TLS verify is default set to `True` for the template, but I wasn't sure if I should default it to `False` to keep it consistent with past code.

Reply:
You can default to False here, and update the spec.yaml with custom `overrides` to enforce `false` as the default.
But this is a temporary work anyway as introducing `use_tls` will change this behavior

Reply:
Sounds good. In that case I will keep it as `True` for now.